doc/man3/flux_attr_get.rst

```diff
@@ -71,5 +71,5 @@ Github: http://github.com/flux-framework
 SEE ALSO
 ========
 
-flux-lsattr(1), flux-getattr(1), flux-setattr(1), flux-broker-attributes(7),
+:man1:`flux-lsattr`, :man1:`flux-getattr`, :man1:`flux-setattr`, :man7:`flux-broker-attributes`,
```

Comment:
There's not much point in listing separate pages for `lsattr`, `getattr`, and `setattr` since they all point to the same man page.